alerts/google-gke/uptime-checks-workload.v1.json

```diff
@@ -0,0 +1,37 @@
+{
+  "displayName": "${CLUSTER_NAME}/${WORKLOAD_NAME} GKE Load Balancer Check uptime failure"
+  "documentation": {},
+  "userLabels": {
+    "workload_name": "${WORKLOAD_NAME}",
```

Comment:
If we want to match all the environment variables set from the Workload Details Observability tab

workload_type = gke_deployment
location = ${LOCATION}
project_id = ${PROJECT_ID}
namespace = ${NAMESPACE}

Reply:
not sure if i'm missing something, where are these user labels getting populated in the workload details tab?